src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+  const cost = days * 40;
+
+  if (days < 3) {
+    return cost;
+  }
+
+  if (days < 7) {
+    return cost - threeDayDiscount;
+  } else {
+    return cost - sevenDayDiscount;
+  }
```

Comment:
redundant else, you can do the final return without it
```suggestion
  }

  return cost - sevenDayDiscount;
```